website/docs/ref/conf.md

```diff
@@ -207,17 +207,24 @@ For example, setting [`compileNamespace`](#compilenamespace) to `window.i18n` cr
 /* eslint-disable */window.i18n={messages: {"..."}}
 ```
 
-## extractBabelOptions
+## extractorParserOptions
 
 Default: `{}`
 
-Specify extra babel options used to parse source files when messages are being extracted. This is required when project doesn't use standard Babel config (e.g. Create React App).
-
-``` json
-{
-  "extractBabelOptions": {
-    "plugins": ["@babel/plugin-syntax-dynamic-import"]
-  }
+Specify extra options used to parse source files when messages are being extracted.
+
+```ts
+"extractorParserOptions": {
+  /**
+   * default true
+   * Use for Stage 3 decorators syntax
+   */
+  decoratorsBeforeExport?: boolean
+  /**
+   * Enable if you use flow. Default false
+   * This will apply Flow syntax to files with .js, cjs, .mjs extension.
+   */
```

Comment:
```suggestion
  /**
   * default false
   * Enable if you use flow. This will apply Flow syntax to files with .js, cjs, .mjs extension.
   */
```